src/assets/i18n/zh-hans.json

```diff
@@ -1,46 +1,46 @@
-  "MSDOSFS Locale": "",
-  "No e-mail address is set for root user or any other local administrator. Please, configure such an email address first.": "",
-  "Open Display Device": "",
-  "PCI device does not have a reset mechanism defined and you may experience inconsistent/degraded behavior when starting/stopping the VM.": "",
-  "Partition": "",
-  "Pool Disks have {alerts} alerts and {smartTests} failed SMART tests": "",
-  "Pool contains {status} Data VDEVs": "",
-  "Pool is not healthy": "",
-  "Pool is using more than {maxPct}% of available space": "",
-  "Pool status is {status}": "",
-  "Root user (not recommended)": "",
+  "Admin Username": "管理员用户名",
+  "Administrative user": "管理用户",
+  "Destination Path": "目的地路径",
+  "Edit Global Configuration": "编辑全局配置",
+  "Edit VM": "编辑 VM",
+  "Filesystem Type": "文件系统类型",
+  "Import Data": "导入数据",
+  "Importing Disk": "导入磁盘",
+  "Importing Disk...": "导入磁盘...",
+  "It may impact currently running services and should be done during planned downtime.": "它可能会影响当前运行的服务，应该在计划停机期间完成。",
+  "MSDOSFS Locale": "MSDOSFS 语言环境",
+  "No e-mail address is set for root user or any other local administrator. Please, configure such an email address first.": "没有为 root 用户或任何其他本地管理员设置电子邮件地址。 请先配置这样一个电子邮件地址。",
+  "Open Display Device": "打开显示设备",
+  "PCI device does not have a reset mechanism defined and you may experience inconsistent/degraded behavior when starting/stopping the VM.": "PCI 设备没有定义重置机制，您可能会在启动/停止 VM 时遇到不一致/性能下降的情况。",
+  "Partition": "分离",
+  "Pool Disks have {alerts} alerts and {smartTests} failed SMART tests": "池磁盘有 {alerts} 个警报和 {smartTests} 个失败的SMART测试",
+  "Pool contains {status} Data VDEVs": "池包含 {status} VDEV 数据",
+  "Pool is not healthy": "池不健康",
+  "Pool is using more than {maxPct}% of available space": "池使用超过 {maxPct}% 的可用空间",
+  "Pool status is {status}": "池状态为 {status}",
+  "Root user (not recommended)": "根用户（不推荐）",
   "Search or enter value": "",
-  "Set up TrueNAS authentication method:": "",
-  "Snapdev": "",
-  "Synchronize Time": "",
-  "Synchronize time": "",
-  "System dataset updated.": "",
-  "The following { n, plural, one {application} other {# applications} } will be upgraded. Are you sure you want to proceed?": "",
-  "This share is configured through TrueCommand": "",
-  "Updating ACL": "",
-  "Upload SSH Key": "",
-  "User account password for logging in to the remote system.": "",
-  "Username on the remote system to log in <b>via Web UI</b> to setup connection.": "",
-  "Username on the remote system which will be used to login <b>via SSH</b>.": "",
+  "Set up TrueNAS authentication method:": "设置 TrueNAS 认证方式：",
+  "Snapdev": "开发快照",
+  "Synchronize Time": "同步时间",
+  "Synchronize time": "同步时间",
+  "System dataset updated.": "系统数据集已更新。",
+  "The following { n, plural, one {application} other {# applications} } will be upgraded. Are you sure you want to proceed?": "以下 { n, plural, one {application} other {# applications} } 将被升级，你确定要继续吗？",
```

Comment:
You can alter expression within { } to match Chinese.
It appears that only one form of the word is used in Chinese for both plural and single things.
If this is true, you can change translation to something like:
```
以下 {n} 应用程式 将被升级，你确定要继续吗？
```

You can test here: http://format-message.github.io/icu-message-format-for-translators/editor.html